### PR TITLE
Make max value and pools mutable

### PR DESCRIPTION
**Pull request by sourabhmarathe**
This PR allows the `admin` to set the pool in the MarketPlace contract and adjust the maximum value allowed per day for a given protocol.

In addition, it formats the contracts for legibility.

### CODE REVIEW

**Review by simonovic86 — DISMISSED** [review]
lgtm!

File: flattened/MarketPlace.flattened.sol [hunk 1]
```diff
@@ -1857,14 +1877,6 @@ contract MarketPlace {
         uint256 m,
         address a
     ) external authorized(admin) returns (bool) {
-        // Verify that the pool has not already been set
-        address pool = pools[u][m];
```
Comment (JTraversa) [inline flattened/MarketPlace.flattened.sol:1861]: LGTM

File: src/Lender.sol [hunk 2]
```diff
@@ -75,7 +75,7 @@ contract Lender {
 
     // Rate limiting protection
     /// @notice maximum amount of value that can flow through a protocol in a day (in USD)
-    uint256 public constant MAX_VALUE = 2_000_000e27;
+    uint256 public maximumValue = 250_000e27;
```
Comment (JTraversa) [inline src/Lender.sol:78]: ✔️

File: src/Lender.sol [hunk 3]
```diff
@@ -315,15 +308,21 @@ contract Lender {
...
+        uint256 p
+    ) external authorized(admin) returns (bool) {
         etherPrice = p;
         return true;
     }
 
+    /// @notice sets the maximum value that can flow through a protocol
+    /// @param m the maximum value by protocol
+    /// @return bool true if the price was set
+    function setMaxValue(uint256 m) external authorized(admin) returns (bool) {
```
Comment (JTraversa) [inline src/Lender.sol:321]: 👍

**Review by JTraversa — APPROVED** [review]
No issues that i can see

**Pull request merged**